spacewalk/certs-tools/spacewalk-certs-tools.changes

```diff
@@ -1,3 +1,6 @@
+- delopy local CA under different name in the truststore to avoid
```

Comment:
Typo "delopy" here and in the other changelog

Reply:
fixed